draft-ietf-capport-architecture.xml

```diff
@@ -827,10 +794,11 @@ o . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . o
      </section>
      <section title="Risk of Nuisance Captive Portal">
        <t>
-         It is possible for any user on the Internet to send signals in attempt
-         to cause the receiving equipment to go to the captive portal. This has been
-         considered and addressed in the following ways:
-         <list>
+         If a Signalling Protocol is implemented, it may be possible for any user on
+         the Internet to send signals in attempt to cause the receiving equipment to
+         go to the captive portal. This has been considered, and implementations may
```

Comment:
Don't say "go to the captive portal"; instead, something like "communicate with the Captive Portal API".